tools/ExtensionLibrary/DevHome.ExtensionLibrary/ViewModels/StorePackageViewModel.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System;
+using System.Threading.Tasks;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using Windows.System;
+
+namespace DevHome.ExtensionLibrary.ViewModels;
+public partial class StorePackageViewModel : ObservableObject
+{
+    [ObservableProperty]
+    private string _productId;
+
+    [ObservableProperty]
+    private string _title;
+
+    [ObservableProperty]
+    private string _publisher;
+
+    [ObservableProperty]
+    private string _packageFamilyName;
+
+    public StorePackageViewModel(string productId, string title, string publisher, string packageFamilyName)
+    {
+        _productId = productId;
+        _title = title;
+        _publisher = publisher;
+        _packageFamilyName = packageFamilyName;
+    }
+
+    [RelayCommand]
+    public async Task LaunchStoreButton(string packageId)
+    {
+        var linkString = $"ms-windows-store://pdp/?ProductId={packageId}&mode=mini";
+        await Launcher.LaunchUriAsync(new (linkString));

```

Comment:
Not necessary right now, but there will be a package installed event in the Extensions Catalog, so Plugin service could update or you could listen for that in the AppExtensionCatalog API to know when to refresh the view after the user installs a plugin while this page is showing.